pkg/sharings/send_mails.go

```diff
@@ -0,0 +1,222 @@
+package sharings
+
+import (
+	"fmt"
+	"net/url"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/instance"
+	"github.com/cozy/cozy-stack/pkg/jobs"
+	"github.com/cozy/cozy-stack/pkg/jobs/workers"
+)
+
+// The skeleton of the mail we will send. The values between "{{ }}" will be
+// filled through the `mailTemplateValues` structure.
+const (
+	mailTemplateEn = `
+        <h2>Hey {{.RecipientName}}!</h2>
+        <p>{{.SharerPublicName}} wants to share a document with you! You will only be able to view it.</p>
+
+        <p>The description given is: {{.Description}}.</p>
+
+        <form action="{{.OAuthQueryString}}">
+            <input type="submit" value="See this document" />
```

Comment:
We discussed to change this description by something like "Click here to accept this sharing", instead of "See this document", which is confusing.